### PR TITLE
Improving printing output

### PR DESCRIPTION
**Pull request by nantonel**
Small modifications to printing output based on old solvers. 

I think it would be nice to have also the cost function value! but I guess this requires adding a new field `ProximalAlgorithm`?

### CODE REVIEW

File: src/algorithms/DouglasRachford.jl [hunk 1]
```diff
@@ -22,7 +22,7 @@ end
 ################################################################################
 # Constructor(s)
 
-function DRSIterator(x0::BlockArray{R}; f=Zero(), g=Zero(), gamma::R=1.0, maxit::I=10000, tol::R=1e-4, verbose=1) where {I, R}
+function DRSIterator(x0::BlockArray{R}; f=Zero(), g=Zero(), gamma::R=1.0, maxit::I=10000, tol::R=1e-4, verbose=2) where {I, R}
```
Comment (lostella) [inline src/algorithms/DouglasRachford.jl]: Why `verbose=2` by default? I would say `1`, that is the minimum (nonzero) verbosity.

File: src/algorithms/DouglasRachford.jl [hunk 2]
```diff
@@ -38,9 +38,23 @@ maxit(sol::DRSIterator) = sol.maxit
 
 converged(sol::DRSIterator, it) = blockmaxabs(sol.FPR_x)/sol.gamma <= sol.tol
 
-verbose(sol::DRSIterator, it) = sol.verbose > 0
+verbose(sol::DRSIterator)     = sol.verbose > 0
+verbose(sol::DRSIterator, it) = sol.verbose > 0 && (sol.verbose == 1 ? true : (it == 1 || it%100 == 0))
```
Comment (lostella) [inline src/algorithms/DouglasRachford.jl]: Related to the above comment: should be `sol.verbose == 2 ? true`. Furthermore, I would make that `100` an option (`output_freq` maybe? Of course defaulting to `100`).

**Comment by nantonel** [comment]
I've made the changes. Also I introduce the `cost`. However, in `ZeroFPR` the `FBE` is printed. I know `FBE` and `f+g` will coincide at minima but wouldn't it be more consistent to print the cost?

File: src/algorithms/DouglasRachford.jl [hunk 3]
```diff
@@ -5,14 +5,16 @@
 # [1] Eckstein, Bertsekas "On the Douglas-Rachford Splitting Method and the Proximal Point Algorithm for Maximal Monotone Operators*", Mathematical Programming, vol. 55, no. 1, pp. 293-318 (1989).
 #
 
-struct DRSIterator{I <: Integer, R <: Real, T <: BlockArray{R}} <: ProximalAlgorithm{I, T}
+mutable struct DRSIterator{I <: Integer, R <: Real, T <: BlockArray{R}} <: ProximalAlgorithm{I, T}
     x::T
     f
     g
+    cost::R
```
Comment (lostella) [inline src/algorithms/DouglasRachford.jl]: Forget about the cost for now, this we will add later. Let's not modify anything in the `initialize` and `iterate` methods in this PR, only visualization stuff.

**Pull request merged**